### PR TITLE
logs user in sentry

### PR DESCRIPTION
**Pull request by amprokop**
untested, need sentry access and the sentry DSN so i can see these coming through

@shanear @orischwartz — are you able to set me up with those?

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -14,6 +15,18 @@ def authenticate
...
   end
 
+  def set_raven_user
+    if ENV['SENTRY_DSN']
+      # Raven sends error info to Sentry.
+      Raven.user_context(
+        id: current_user.id,
+        email: current_user.email,
+        ip_address: current_user.ip_address,
+        station_id: station_id
```
Comment (orischwartz) [inline app/controllers/application_controller.rb]: Should this be `station_id: current_user.station_id` ? Where is the `station_id` value coming from?

**Comment by shanear** [comment]
tested it. When i fix the bug that Ori pointed out, it works 😎 

this is what shows up in sentry. very cool
screen shot 2016-10-26 at 2 45 33 pm

**Comment by amprokop** [comment]
cool, updated to fix the pointer @shanear

**Comment by shanear** [comment]
@orischwartz this is good to go from my perspective

**Pull request merged**